Packs/AlienVault_USM_Anywhere/Integrations/AlienVault_USM_Anywhere/AlienVault_USM_Anywhere.py

```diff
@@ -136,6 +137,24 @@ def get_time_range(time_frame=None, start_time=None, end_time=None):
     return date_to_timestamp(start_time), date_to_timestamp(end_time)
 
 
+def convert_timestamp_to_iso86(timestamp: str, timezone_letter: str = 'Z') -> str:
```

Comment:
did you check if we can use `dateparser.parse` instead ?
see https://dateparser.readthedocs.io/en/latest/#timezone-and-utc-offset